.gitignore

```diff
@@ -115,6 +115,11 @@ venv.bak/
 # compiled protos
 *pb2.py
 
+#Json folder
```

Comment:
you said its in progress so just placing this comment here so we don't forget since this will be a living PR.

we should stop using `cwd` and place locally created datasets into the RMLCache, then we can remove these gitignore paths